### ISSUE TITLE
[Bug]: Favorite star should be bright yellow

### ISSUE THREAD

**Issue by lonix1** [FreeTubeApp/FreeTube#2807]
### Guidelines

- [X] I have encountered this bug in the [latest release of FreeTube](https://github.com/FreeTubeApp/FreeTube/releases).
- [X] I have searched the [issue tracker](https://github.com/FreeTubeApp/FreeTube/issues) for open and closed issues that are similar to the bug report I want to file, without success.
- [X] I have searched the [documentation](https://docs.freetubeapp.io/) for information that matches the description of the bug I want to file, without success.

### Describe the bug

Channels:
1. open a channel
2. each video shows a favourite/star icon in its top right corner
3. star is light grey or light yellow

Videos:
1. open a video
2. it shows a favourite/star icon under the video
3. star is light grey or light yellow

It's hard to distinguish between light grey and light yellow - I often can't tell whether the video is favourited or not.

I checked on multiple monitors, the problem is the same.

### Expected Behavior

Please use a brighter / darker yellow.

### Issue Labels

accessibility issue, usability issue, visual bug

### FreeTube Version

v0.18.0 Beta

### Operating System Version

Ubuntu 22.04.1 LTS

### Installation Method

.deb

### Primary API used

Local API

### Nightly Build

- [ ] I have encountered this bug in the latest [nightly build](https://docs.freetubeapp.io/development/nightly-builds).

**Comment by efb4f5ff-1298-471a-8973-3d47447115dc** [comment]
This isnt a bug. It was the same color in the previous version

v0.17.1
![v0 17 1](https://user-images.githubusercontent.com/73130443/199838758-6caead9c-f79d-4032-86dc-9a3cb171d2e3.png)

v0.18.0
![v0 18 0](https://user-images.githubusercontent.com/73130443/199838761-8c155a25-91c9-453e-a5e7-a196378d0bb9.png)

The request of color change of the star is a duplicate of https://github.com/FreeTubeApp/FreeTube/issues/1762

**Comment by cocoonkid** [comment]
Yes, longtime big issue here. barely visible.
